charts/accurate/MIGRATION.md

```diff
@@ -0,0 +1,100 @@
+# Migrate from kustomize to Helm
+
+This document describes the steps to migrate from kustomize to Helm.
+
+## Install Helm chart
+
+There is no significant difference between the manifests installed by kusomize and those installed by Helm.
+
+If a resource with the same name already exists in the Cluster, Helm will not be able to create the resource.
+
+```console
+$ helm repo add accurate https://cybozu-go.github.io/accurate/
+$ helm repo update
+$ helm install --namespace accurate accurate accurate/accurate
+Error: rendered manifests contain a resource that already exists. Unable to continue with install: ServiceAccount "accurate-controller-manager" in namespace "accurate" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "accurate"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "accurate"
+```
+
+Before installing Helm chart, you need to manually delete the resources.
+
+```console
+$ helm template --namespace accurate accurate accurate/accurate | kubectl delete -f -
+serviceaccount "accurate-controller-manager" deleted
+clusterrole.rbac.authorization.k8s.io "accurate-manager-role" deleted
+clusterrole.rbac.authorization.k8s.io "accurate-subnamespace-editor-role" deleted
+clusterrole.rbac.authorization.k8s.io "accurate-subnamespace-viewer-role" deleted
+clusterrolebinding.rbac.authorization.k8s.io "accurate-manager-admin" deleted
+clusterrolebinding.rbac.authorization.k8s.io "accurate-manager-rolebinding" deleted
+role.rbac.authorization.k8s.io "accurate-leader-election-role" deleted
+rolebinding.rbac.authorization.k8s.io "accurate-leader-election-rolebinding" deleted
+service "accurate-webhook-service" deleted
+deployment.apps "accurate-controller-manager" deleted
+certificate.cert-manager.io "accurate-serving-cert" deleted
+issuer.cert-manager.io "accurate-selfsigned-issuer" deleted
+mutatingwebhookconfiguration.admissionregistration.k8s.io "accurate-mutating-webhook-configuration" deleted
+validatingwebhookconfiguration.admissionregistration.k8s.io "accurate-validating-webhook-configuration" deleted
+Error from server (NotFound): error when deleting "STDIN": configmaps "accurate-config" not found # This is because the ConfigMap created by ConfigMapGeneraor will be suffixed. There is no problem to ignore it.
+```
+
+Then install Helm chart again.
+
+```console
+$ helm install --namespace accurate accurate accurate/accurate
+NAME: accurate
+LAST DEPLOYED: Fri Aug 20 10:12:03 2021
+NAMESPACE: accurate
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+### Release Name
+
```

Comment:
This was prepared to explain that the name may be changed when migrating from kusotomize, but it is no longer necessary since the flow is now to delete what was once created by kustomize.

Reply:
I will be remove it.

Reply:
removed: https://github.com/cybozu-go/accurate/pull/11/commits/cc0e5dd22ff18a364be08a61aeb847af0d9f7040